packages/marshal/src/helpers/remotable.js

```diff
@@ -105,7 +109,10 @@ const checkRemotableProtoOf = (original, check) => {
   // would always return true.
   // @ts-ignore TypeScript assumes what we're trying to check
   if (proto === objectPrototype) {
-    return reject(X`Remotables must be explicitly declared: ${q(original)}`);
+    return (
+      reject &&
+      reject(X`Remotables must be explicitly declared: ${q(original)}`)
```

Comment:
Weird that you rescue your `reject` refactoring here but revert it elsewhere. How about rescuing it everywhere? That'll bring us closer to your other refactoring suggestion, where we parameterize these with an optional `reject` directly.

Reply:
Good point; I don't know why I didn't go back to the other files! Done.